Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -126,6 +129,9 @@ async function run() {
             await DeployUsingKuduDeploy(webDeployPkg, azureWebAppDetails, publishingProfile, virtualApplication, isFolderBasedDeployment, takeAppOfflineFlag);
 
         }
+        if(scriptType) {
```

Comment:
Yes, by default, it is empty string. If condition validates it and returns false.